torchvision/prototype/datapoints/_image.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import annotations
 
-from typing import Any, List, Optional, Tuple, Union
+from typing import Any, List, Literal, Optional, Tuple, Union
```

Comment:
This only works for Python >= 3.8

```suggestion
from typing import Any, List, Optional, Tuple, Union

from typing_extensions import Literal
```